bbq/vm/context.go

```diff
@@ -246,3 +246,10 @@ func (c *Context) GetMethod(
 		method,
 	)
 }
+
+func (c *Context) GetFunction(
+	location common.Location,
+	name string,
+) interpreter.FunctionValue {
+	return c.lookupFunction(location, name)
```

Comment:
Given that it forwards the call as-is, could we just inline `lookupFunction` here and replace all callers of `lookupFunction` with `GetFunction`?

Reply:
yah, that makes sense. Added in https://github.com/onflow/cadence/pull/3933/commits/92bcca9567b6c92f5e52776144fb7bd077750ad9